### DISCUSSION TITLE
How to convert fine-tuned model to GGUF ?

### DISCUSSION THREAD

**Discussion by heyili** [ggml-org/llama.cpp#4997] [Q&A]
Hey guys, 
I am trying to convert a fine-tuned model from huggingface `jzdesign/falcon-finetune-midjourney-falcon` `.GGUF` format. The fine-tuned one only contains a `.json` file that points to the base model and a `.bin` file 
which contains all the weights. 

my folder structure:
\fine_tune
   -adapter_model.bin
   -adapter_model.json

When I try to run 
```
!python llama.cpp/convert.py fine_tune/ \
  --outfile fine_tune_convert \
  --outtype q8_0
```
I get 
```
/content/llama.cpp/gguf-py
Traceback (most recent call last):
  File "/content/llama.cpp/convert.py", line 1658, in <module>
    main(sys.argv[1:])  # Exclude the first element (script name) from sys.argv
  File "/content/llama.cpp/convert.py", line 1577, in main
    model_plus = load_some_model(args.model)
  File "/content/llama.cpp/convert.py", line 1343, in load_some_model
    raise Exception(f"Can't find model in directory {path}")
Exception: Can't find model in directory fine_tune
```
So far I followed the tutorial from [https://github.com/ggerganov/llama.cpp/discussions/2948]

In the end, what I would like to have, is a file like `falcon-finetune-midjourney-falcon.gguf`

Does someone know how to solve this problem ?

Thank you!

**Comment by asorteberg** [comment]
I am also seeking clarification on this process. Additionally, how can we use safetensor adapter files rather then bin files to reach the same conclusion.
